equinix/data_source_fabric_port_acc_test.go

```diff
@@ -1,33 +1,17 @@
 package equinix_test
 
 import (
-	"encoding/json"
 	"fmt"
 	"github.com/equinix/equinix-sdk-go/services/fabricv4"
-	"os"
+	"github.com/equinix/terraform-provider-equinix/internal/fabric/testing_helpers"
 	"testing"
 
 	"github.com/equinix/terraform-provider-equinix/internal/acceptance"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 )
 
-const (
-	FabricDedicatedPortEnvVar = "TF_ACC_FABRIC_DEDICATED_PORTS"
-)
-
-type EnvPorts map[string]map[string][]fabricv4.PortResponse
-
-func GetFabricEnvPorts(t *testing.T) EnvPorts {
```

Comment:
The removal of this function is causing a couple errors in the unit test workflow: https://github.com/equinix/terraform-provider-equinix/actions/runs/9375209054/job/25812858456?pr=693#step:6:32

It looks like there are still a couple references to this function; you could update those here if you like, but if you want to minimize the diff for this PR, you could also keep this function in place and continue to use it within the `equinix` package.  If you keep it, I recommend adding a `// Deprecated: <details>` comment that directs the reader to prefer the new, equivalent function in `testing_helpers`.